temporalio/activity.py

```diff
@@ -135,6 +138,29 @@ def _logger_details(self) -> Mapping[str, Any]:
 _current_context: contextvars.ContextVar[_Context] = contextvars.ContextVar("activity")
 
 
+@dataclass
+class ActivityCancellationDetails:
+    """Provides the reasons for the activity's cancellation"""
+
+    not_found: bool = False
+    cancelled: bool = False
```

Comment:
I think this should be `cancel_requested`. All of these represent "cancelled" in a way, but only this one is due to cancel request. I know Core doesn't differentiate server-cancelled from activity-cancelled in the name, but I am thinking lang may want to.